central/image/datastore/internal/store/postgres/index.go

```diff
@@ -20,6 +19,7 @@ func init() {
 	mapping.RegisterCategoryToTable(v1.SearchCategory_IMAGES, schema)
 }
 
+// NewIndexer returns a new image indexer.
```

Comment:
Seems like you can just add this to the generator instead of removing the hook to code gen

Reply:
Good for other stores but I am skipping auto-gen for images entirely. The table name is must have param so can't conditionally not generate store unless I add another param explicitly to indicate gen store.